docker-hub/billing/add-seats.md

```diff
@@ -4,45 +4,47 @@ keywords: Docker, Docker Hub, subscription, update, add, seats
 title: Add seats to your plan
 ---
 
-You can manage your Docker Hub subscription anytime by adding seats to your plan. If you add seats to your plan in the middle of your billing cycle, a prorated amount will be charged for the additional seats.
+You can manage your Docker Hub subscription anytime by adding seats to your plan.
 
 ## Add seats to a monthly plan
 
 > **Note:**
 >
-> You must be a member of the ‘Owners’ team to add seats to your organization.
+>- A monthly subscription includes monthly seats only.
+>- You must be a member of the ‘Owners’ team to add seats to your organization.
+>- When you add seats to your monthly plan in the middle of your billing cycle, you are charged a prorated amount for the additional seats.
 
 To add seats to a monthly plan:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
-2. Navigate to **Organizations** from the menu at the top of the page and then choose your organization.
+2. Navigate to **Organizations** from the menu at the top of the page and select your organization.
 
 3. Select the **Billing** tab and click **Add seats**.
 
-4. Specify the number of seats you’d like to add and click **Purchase**.
+4. Specify the number of seats you want to add and click **Purchase**.
 
-5. The **Billing** tab displays the number of seats currently in use.
+5. The **Billing** tab gives you the number of seats currently in use.
```

Comment:
I think "displays" is still correct in this context. Was there any specific reason for this change?

Reply:
@usha-mandya Sure, I just reverted it back to displays. Initially it sounded like the passive voice.